source/tutorials/openmm.rst

```diff
@@ -17,4 +23,129 @@ folder of the GitHub repository. It contains:
+Within this tag, you'll find:
+
+    **Particles:** They are represented with ``<Particle>`` tags, each containing:
+
+        - Mass
+        - Charge (if using a `NonbondedForce`)
+
+    **Forces:** Various force components are represented by specific tags (a detailed list could be found `here <http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html#standard-forces>`_), for example:
+
+        - ``<HarmonicBondForce>``: For bond stretching
+        - ``<HarmonicAngleForce>``: For angle bending
+        - ``<PeriodicTorsionForce>``: For dihedral angles
+        - ``<NonbondedForce>``: For electrostatic and van der Waals interactions
+        - ``<CustomNonbondedForce>``: For user-defined nonbonded interactions
+
+        Each force tag contains parameters specific to that force type. For example:
+
+        .. code-block:: xml
+
+           <HarmonicBondForce>
+             <Bond p1="0" p2="1" length="0.1" k="1000"/>
+           </HarmonicBondForce>
+
+    **Constraints:** If present, constraints are listed under a ``<Constraints>`` tag:
+
+        .. code-block:: xml
+
+           <Constraints>
+             <Constraint p1="0" p2="1" distance="0.1"/>
+           </Constraints>
+
+
+3. **OpenMM serialized integrator**:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``<Integrator>`` tag contains parameters that specify the integration method to be used to simulate dynamics, such as the type of integrator, simulation time step and temperature:
+
+.. code-block:: xml
+
+    <Integrator type="LangevinIntegrator" constraintTolerance="1e-05" friction="4" randomSeed="0" stepSize=".0005" temperature="300" version="1" />
+
+More details on integrators can be found `here <http://docs.openmm.org/latest/userguide/theory/04_integrators.html>`_.
+
+Usage
+-----
+
+The :mod:`nanover.openmm.serializer` module provides the ``serialize_simulation`` and ``deserialize_simulation`` functions which allow saving and loading OpenMM simulations to/from XML files. The serialization captures the complete simulation state including:
+
+- Structure coordinates and topology (as PDBx/PDB)
+- OpenMM System definition
+- Integrator configuration
+
+Serializing a simulation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To save a simulation to an Openmm XML::
+
+    xml_string = nanover.openmm.serializer.serialize_simulation(simulation)
+
+    with open("sim.xml", "w") as f:
+        f.write(xml_string)
+
+Deserializing a simulation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To load a simulation from an Openmm XML::
+
+    with open("sim.xml", "r") as f:
+        simulation = nanover.openmm.serializer.deserialize_simulation(f.read())
+
+The ``deserialize_simulation`` function accepts optional arguments:
+
+- ``imd_force``: A CustomExternalForce for interactive molecular dynamics
+- ``platform_name``: The OpenMM platform to use (e.g. "CUDA", "OpenCL")
+
+For example::
+
+    simulation = nanover.openmm.serializer.deserialize_simulation(
+        xml_string,
+        platform_name="CUDA"
+    )
+
```

Comment:
May be worth linking to an OpenMM tutorial that demonstrates how to write OpenMM input files (e.g. the polyalanine notebook in the OpenMM examples directory)

Reply:
Is [this](https://github.com/openmm/openmm-cookbook/blob/main/notebooks/cookbook/Saving%20Systems%20to%20XML%20Files.ipynb) what you mean?

Reply:
I think Harry means the NanoVer OpenMM tutorial about polyanalanine: https://github.com/IRL2/nanover-protocol/blob/main/examples/openmm/openmm_polyalanine.ipynb

Reply:
resolved in commit [e43e5bf](https://github.com/IRL2/nanover-docs/pull/115/commits/e43e5bf8a8b24a180e5f9e6fa04ecfc6947ae248)